pkg/api/azenvtypes.go

```diff
@@ -173,6 +173,15 @@ var (
 		ImageVersion:   "2020.05.13",
 	}
 
+	// AKSGPU1804OSImageConfig is the AKS image with GPU drivers and device plugin based on Ubuntu 18.04-LTS.
+	// TODO: Update sku and version after GPU image is actually published
+	AKSGPU1804OSImageConfig = AzureOSImageConfig{
+		ImageOffer:     "aks",
+		ImageSku:       "aks-gpu-1804-202005",
```

Comment:
can we follow the pattern aks-ubuntu-1804-202005, maybe call it aks-ubuntu-1804-gpu-202005?

Reply:
made it aks-ubuntu-gpu-1804 so numbers go in the end